device-types/QNAP/TS-432PXU.yaml

```diff
@@ -0,0 +1,38 @@
+---
+manufacturer: QNAP
+model: TS-432PXU
+slug: qnap-ts-432pxu
+part_number: TS-432PXU
+u_height: 1
+airflow: front-to-rear
```

Comment:
To have the most complete data possible could you add the `weight` and `weight-unit` values

Reply:
Addressed in https://github.com/netbox-community/devicetype-library/pull/2521/commits/2e7c8da0c02c2b630fe4583e36af1b60ee9702b3